composer.json

```diff
@@ -4,15 +4,17 @@
     "keywords": ["event-loop", "asynchronous"],
     "license": "MIT",
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "evenement/evenement": "^3.0||^2.0"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.8"
     },
     "suggest": {
         "ext-libevent": ">=0.1.0",
         "ext-event": "~1.0",
-        "ext-libev": "*"
+        "ext-libev": "*",
+        "ext-pcntl": "For signals support"
```

Comment:
Afaict this is only need for the default loop, right? Documentation should probably be updated to explicitly say *when* this extension is recommended?